packages/storybook/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "storybook",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "start-storybook -p 6006",
+    "watch": "yarn start",
+    "foo": "concurrently --no-color -n snapshot,e2e \"yarn test:snapshot\" \"yarn test:e2e\"",
```

Comment:
it probably needs another name

Reply:
yeah it was named test before, but then it got triggered on each commit which is unnecessary I think.

Reply:
what's about `test:all`?